src/templates/_common/components/mt5_banner.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 const mt5Banner = () => (
-    <a className='mt5_banner' data-show='-eucountry' target='_blank' rel='noopener noreferrer' href='https://deriv.com/interim/dmt5/?utm_source=binary&utm_medium=referral&utm_campaign=deriv-launch&utm_content=binary-mt5-page'>
+    <a className='mt5_banner' data-show='-eucountry' target='_blank' rel='noopener noreferrer' href={`https://${window.location.hostname.split('.').splice(1).join('.') || 'deriv.com'}/interim/dmt5/?utm_source=binary&utm_medium=referral&utm_campaign=deriv-launch&utm_content=binary-mt5-page`}>
```

Comment:
Let's create a sharing function for this since it is being used in multiple places!

Reply:
Okay sure